### PR TITLE
Wishlist and product pages

### PR DESCRIPTION
**Pull request by veereshbv04**
Made wishlist and product listing page, Please review and suggest improvements

Live link : [link](https://bookstorecss.netlify.app/pages/home.html)

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -122,6 +125,7 @@ img{
     color: green;
```
Comment (abhisheksahani-nova) [inline styles/index.css]: use css variables throughout the app

**Review by abhisheksahani-nova — APPROVED** [review]
Code looks good, great work

**Review by humzahasan — APPROVED** [review]
Suggestion : Code looks good to merge. Just a suggestion to have some basic animation such as hover on card or on the footer link to make it more interactive.

**Comment by veereshbv04** [comment]
> Suggestion : Code looks good to merge. Just a suggestion to have some basic animation such as hover on card or on the footer link to make it more interactive.

Yeah i will add it, Thank You

**Pull request merged**